machine/delta/delta_wb2448/i2ctools/i2ctools-sys-eeprom-platform.patch

```diff
@@ -0,0 +1,17 @@
+Add platform specific stuff for the sys-eeprom
```

Comment:
Using i2ctools to support system EEPROM access has been obsolete since March 2015.  See this email on the mailing list:
http://lists.opencompute.org/pipermail/opencompute-onie/2015-March/000512.html

Use the busybox implementation instead.

As an example, again consult this Agema commit: b1d7f72bcf85cfec9cc5dfb52890e889fe258195